liota/dccs/iotcc.py

```diff
@@ -150,6 +133,10 @@ def on_response(msg):
 
             return RegisteredEntity(entity_obj, self, self.reg_entity_id)
 
+    def _check_version(self, json_msg):
+	if json_msg["version"] != self._version:
```

Comment:
Recommend making version an integer and allowing yourself to support a range. A good integer version number is YYYYMMDDxx -- new versions will always be a bigger value... you don't get major.minor.dot but it works pretty well and is a fast parse and comparison.